src/application-intl/ApplicationIntlContext.jsx

```diff
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+import { intlShape } from 'react-intl';
+
+const ApplicationIntlContext = createContext();
+
+const contextShape = intlShape;
+
+export default ApplicationIntlContext;
+export { contextShape };
```

Comment:
I'm surprised the linter isn't complaining about it needing a new line at the end of the file

Reply:
They're present in this file and throughout, just not rendered here.